webpack.config.js

```diff
@@ -47,6 +49,22 @@ const config = [
       }
     })
     .getConfig(),
+  // Bootstrap components - can't just rely on CopyWebpackPlugin for these because some of them require
+  // additional files that aren't declared in the documentation.
+  new JavascriptWebpackConfig('bootstrap', PATHS, 'silverstripe/admin')
+    .setEntry(glob.sync(`${PATHS.MODULES}/bootstrap/js/dist/**/*.js`).reduce((obj, el) => {
+      const parsedPath = path.parse(el);
+      const dir = parsedPath.dir.replace(new RegExp(`${PATHS.MODULES}\/bootstrap\/js\/dist\/?`), '');
+      obj[path.join(dir, parsedPath.name)] = el;
+      return obj;
+    }, {}))
+    .mergeConfig({
+      output: {
+        path: `${path.resolve('thirdparty')}/bootstrap/js/dist`,
+        filename: '[name].js',
+      },
+    })
+    .getConfig(),
```

Comment:
These files weren't actually getting updated when bootstrap was updated! Now they will be.